pandas/src/inference.pyx

```diff
@@ -20,6 +20,20 @@ cdef extern from "headers/stdint.h":
     enum: INT64_MAX
     enum: INT64_MIN
 
+ctypedef fused int_dtype:
```

Comment:
move to src/util.pxd (and maybe call ``int_like``) (and put a comment on the purpose)